packages/perseus/src/perseus-types.ts

```diff
@@ -187,6 +187,8 @@ export type SimpleMarkdownTesterWidget = Widget<'simple-markdown-tester', Perseu
 export type UnitInputWidget = Widget<'unit-input', PerseusUnitInputWidgetOptions>;
 // prettier-ignore
 export type VideoWidget = Widget<'video', PerseusVideoWidgetOptions>;
+//prettier-ignore
+export type AutoCorrectWidget = Widget<'auto-correct', any>;
```

Comment:
I noticed in a comment later on that you mentioned some of these `any`s are because `auto-correct` can fill in for any other widget. If that's the case here, maybe it could use `PerseusWidgetOptions` which ORs together all of our existing widget options.